dockerfiles/centos-7-pg10/scripts/fetch_and_build_rpm

```diff
@@ -1,7 +1,16 @@
 #!/bin/bash
 
 # make bash behave
-set -euo pipefail
+set -eo pipefail
```

Comment:
When initializing the docker image in OneBranch, docker run process fall into this block so it does not die